fairroot/fastsim/CMakeLists.txt

```diff
@@ -58,5 +58,21 @@ fairroot_target_root_dictionary(${target}
   EXTRA_INCLUDE_DIRS ${CMAKE_INSTALL_FULL_INCLUDEDIR}
 )
 
+# exchange linker flag --enable-new-dtags by --disable-new-dtags
+# to use RPATH instead of RUNPATH in the generated library. When using
+# RUNPATH on many Linux systems the indirect library dependencies are not
+# found when linking an executable which uses the FairFastSim library and
+# such that the linking fails. This bug somehow depends on the Linux system
+# and the linker version.
+# No better solution was found so far. In principle there should be a way
+# such that also when using RUNPATH the indirect dependencies are properly
+# found.
+# Send patches to upstrean such that that geant4, geant4_vmc and vgm
+# correctly build libraries with RPATH information.
```

Comment:
:+1: Opened https://github.com/FairRootGroup/FairSoft/issues/535 to track this effort!